### ISSUE TITLE
v10 website_product_configurator

### ISSUE THREAD

**Issue by max3903** [pledra/odoo-product-configurator#24]
Using version 10, I have configured a desktop with different choice of memory.

Memory is a product with 4 variants and different prices:
```
Memory	Memory: 4 Gb	100.00 $
Memory	Memory: 8 Gb	110.00 $
Memory	Memory: 16 Gb	120.00 $
Memory	Memory: 32 Gb	130.00 $
```

On the website, when I set the memory to 4 Gb, I get 100 $ added to the total on the right.

If I decide to change to 8 Gb, the total is unchanged. The right column and the total should be updated with 110 $.

**Comment by PCatinean** [comment]
@max3903 I think this is a common misconception, happened a number of times already.

How I see options in configurable products:

1. It's either a filter and does not alter the final product. It just limits other options that can be other filters or products. 

For example 16GB RAM would be a filter which removes processors and motherboards not supporting that amount.

2. It's a product/service that changes the composition of the final product or the way it's manufactured, i.e 16 GB Kingston 2111mhz would be an example of a product option

Following this logic since only product options increase the final price of the configurable products, it makes no sense to store image/price information on the attribute value itself.

This is why extra prices are added for attribute values with products linked to them.

Also the images of the options are extracted from the product and retrieving the price from products via pricelist offers much more flexibility. Not to mention it also serves as a bom for manufacturing and deducts products from stock

Long story short, create a regular product, give it a price and link it to the option (attribute.value)

I do have to do something to avoid confusion:

Remove the extra price field from attribute value views, make it a computed field getting price from product or something similar.

There is also info about this in the documentation: https://www.pledra.com/odoo-product-configurator/documentation/

I am btw aware this needs work and I will do my best to update it it's just that I need extra time since I do the full maintenance of the repo and do not share the workload with a third party.

Hope this answers your question.

**Comment by max3903** [comment]
@PCatinean Thanks for the answer. I was expecting another behavior but we will be able to change it with an additional module when necessary. Have a great day!
